example-script-runtime-service/project/project.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
this is one of the changes with the upgrade in m6, we won't need this file. We can remove it from here, and i need to remove it later from the other projects except process-runtiime